test/splitWhenever.js

```diff
@@ -0,0 +1,17 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+
+describe('splitWhenever', function() {
```

Comment:
I'd like to see a few more tests in here. Specifically, we should be testing what happens with an empty array, as well as where the first or last element matches the predicate, as well as when no element of the list matches the predicate.